src/components/common/BottomSheet/BottomSheet.tsx

```diff
@@ -2,19 +2,42 @@ import { motion } from 'motion/react';
 import Icon from '../Icon';
 import { useBottomSheet } from './hooks';
 
+type BottomSheetVariant = 'default' | 'centered';
```

Comment:
default보다는 left가 좋아보여요!
default가 어느 방향인지 알 수가 없어서 의견드립니당

```suggestion
type BottomSheetVariant = 'left' | 'center';
```